### PR TITLE
Clean up generator script

### PR DESCRIPTION
**Pull request by ryanplusplus**

### CODE REVIEW

File: bin/generator.sh [hunk 1]
```diff
@@ -1,18 +1,18 @@
 #!/bin/bash
 
-# Used to start the generator 
+# Used to start the generator
 
 # error prints an error to STDERR and exits.
 
 if [ -z "$1" ] ; then
-    echo "usage: ./bin/template.sh exercise-slug "
```
Comment (ryanplusplus) [inline bin/generator.sh:8]: `template.sh` isn't the name of the script

File: bin/generator.sh [hunk 2]
```diff
@@ -1,18 +1,18 @@
...
+    echo "usage: $0 exercise-slug "
     exit 1
 fi
 
 slug="$1"
 
 if test ! -d "./bin/templates"; then
-  mkdir "./bin/templates"
+    mkdir "./bin/templates"
```
Comment (ryanplusplus) [inline bin/generator.sh:15]: Inconsistent indentation

File: bin/generator.sh [hunk 3]
```diff
@@ -1,18 +1,18 @@
 #!/bin/bash
 
-# Used to start the generator 
+# Used to start the generator
```
Comment (ryanplusplus) [inline bin/generator.sh:3]: Trailing whitespace

**Review by meatball133 — APPROVED** [review]
Nice!✨

These are kinda small changes, but doing any change that can affect how the script works, do I want to be disscused with me in beforehand. Just to let you know.

**Pull request merged**